src/mudblazor/MudBlazor.Template/Program.Main.cs

```diff
@@ -74,7 +75,7 @@ public static void Main(string[] args)
         #endif
         builder.Services.AddDatabaseDeveloperPageExceptionFilter();
 
-        builder.Services.AddIdentityCore<ApplicationUser>(options => options.SignIn.RequireConfirmedAccount = true)
+        builder.Services.AddIdentityCore<ApplicationUser>(options => options.SignIn.RequireConfirmedAccount = false)
```

Comment:
This change fixes the "Error: Invalid login attempt." message that users see when they try to log in after registering.  What's happening is that the site is requiring confirmation, but at the same time has no email provider configured.  Thus, users would have to manually connect to the database, update their record, then sign in.  Quite a pain.  With this as `false`, users are immediately signed in upon registering.  My only concern is that it's less secure.

Reply:
I've restored this line to `true`  (require confirmation).   If people complain then we can change this back.